README.md

```diff
@@ -92,9 +92,10 @@ additional metrics!
 > **collectors**
 > * v2alpha1/cronjob
 >
-> If users want to enable this feature when kubernetes version larger than 1.7,
-> It must be configured, with the following parameter setting for apiserver.
-> `--runtime-config=batch/v2alpha1=true`
+> If users want to enable this feature in kubernetes version less than 1.8,
+> it must be configured with the following parameter setting for apiserver:
+> `--runtime-config=batch/v2alpha1=true`. For cluster versions >= 1.8, cronjobs
+> are enabled by default in batch/v2alpha1
```

Comment:
I think we should just remove this statement all together, as kube-state-metrics only supports using the v1beta1 API.

Reply:
Sounds good, I left the following lines though: https://github.com/kubernetes/kube-state-metrics/pull/411/files#diff-04c6e90faac2675aa89e2176d2eec7d8R100

Since it seems they still apply to the GPU metrics.

Reply:
According to Kubernetes 1.10 release note:

> The alpha Accelerators feature gate is deprecated and will be removed in v1.11. Please use device plugins (https://github.com/kubernetes/features/issues/368) instead. They can be enabled using the DevicePlugins feature gate. (#57384, @mindprince)

gpu support for kube-state-metrics will exist at least until Kubernetes 1.10. It's ok to keep gpu related statement here. 

@r0fls BTW, could you please update the statement about the deprecation for gpu since it is not accurate for now:
`are in alpha stage and will be deprecated when the kubernetes gpu support is final in 1.9 version.` -> `are in alpha stage and will be deprecated when the kubernetes accelerator feature support is final in 1.11 version.`

Reply:
Others lgtm.

Reply:
✅ done

Reply:
lgtm.

Waiting for a secondary review from @brancz